src/main/java/com/singlestore/jdbc/plugin/credential/browser/TokenWaiterServer.java

```diff
@@ -192,6 +192,7 @@ public DummyAlgorithm(String name) {
     @Override
     public void verify(DecodedJWT decodedJWT) throws SignatureVerificationException {}
 
+    @Deprecated
```

Comment:
I think that deprecated is the wrong tag for this function as it is still actively used. 
This function is added to satisfy an interface. Probably, it is better to ignore the warning.